_data/retail.yml

```diff
@@ -64,6 +64,8 @@ websites:
       url: https://www.bebivodka.com/
       twitter: vegasbebi
       img: bebivodka.png
+      country: US
+      city: Las Vegas
```

Comment:
note: We also have a state: tag that we can use at the same time if you're doing this, and I believe we are looking for 2 letter lower case country codes, but I might be wrong

Reply:
appears uper and lower case dont wmatter